lib/logstash/config/mixin.rb

```diff
@@ -121,13 +123,17 @@ def config_name(name=nil)
       return @config_name
     end
 
+    # Deprecated: Declare the version of the plugin
+    # inside the gemspec.
     def plugin_status(status=nil)
       milestone(status)
     end
 
+    # Deprecated: Declare the version of the plugin
+    # inside the gemspec.
     def milestone(m=nil)
```

Comment:
The parameter is needed but I could remove the `=nil` since we are not using this method as a getter.